test/unit/presenters/document_list_export_presenter_test.rb

```diff
@@ -58,4 +59,25 @@ class DocumentListExportPresenterTest < PresenterTestCase
     assert_equal('Org 1', pr.lead_organisations)
   end
 
+  test '#policies returns a list of the related policies' do
+    policy = create(:policy)
+    news   = create(:news_article, related_documents: [policy.document])
+    pr = DocumentListExportPresenter.new(news)
+    assert_equal [policy.title], pr.policies
+  end
+
+  test '#policies returns policy titles with future-policies flag on' do
+    stub_content_register_policies
+    future_policies_setting = FeatureFlag.enabled?('future_policies')
+    FeatureFlag.find_or_create_by(key: 'future_policies')
+    FeatureFlag.set('future_policies', true)
+
+    begin
+      news = create(:news_article, policy_content_ids: [policy_1["content_id"]])
+      pr = DocumentListExportPresenter.new(news)
+      assert_equal [policy_1["title"]], pr.policies
+    ensure
+      FeatureFlag.set('future_policies', future_policies_setting)
```

Comment:
This is no longer needed as the flags are now stored in the DB rather than in memory, so the transactions clean up for us.